common/beerocks/bwl/dummy/ap_wlan_hal_dummy.cpp

```diff
@@ -177,13 +177,19 @@ bool ap_wlan_hal_dummy::update_vap_credentials(
     std::list<son::wireless_utils::sBssInfoConf> &bss_info_conf_list)
 {
     for (auto bss_info_conf : bss_info_conf_list) {
-        LOG(DEBUG) << "Received credentials for ssid: " << bss_info_conf.ssid << " auth_type: "
```

Comment:
Why did you move the DEBUG print after the validations?
If something goes wrong you won't have any prints at all... The function simply returns `false`